### PR TITLE
Fixed Sobol' indices examples

### PR DESCRIPTION
**Pull request by mbaudin47**
This PR considers the Sobol' sampling examples
* simplifies the Ishigami formula with the power operator
* removes the unnecessary independent copula (which is independent, by default)
* provides examples for the second order indices, removing the unnecessary computations when they are not required

### CODE REVIEW

File: python/src/MartinezSensitivityAlgorithm_doc.i.in [hunk 1]
```diff
@@ -72,19 +72,43 @@ SobolIndicesAlgorithm
...
+>>> inputDesign = ot.SobolIndicesExperiment(distribution, size).generate()
+>>> outputDesign = model(inputDesign)
+>>> # sensitivity analysis algorithm
+>>> sensitivityAnalysis = ot.MartinezSensitivityAlgorithm(inputDesign, outputDesign, size)
+>>> print(sensitivityAnalysis.getFirstOrderIndices())
+[0.30449,0.448506,-0.0711394]
+>>> print(sensitivityAnalysis.getTotalOrderIndices())
+[0.543004,0.29501,0.304615]
+
+Estimate first, total and second order Sobol' indices:
```
Comment (jschueller) [inline python/src/MartinezSensitivityAlgorithm_doc.i.in]: you need to skip a line here

File: python/src/MartinezSensitivityAlgorithm_doc.i.in [hunk 2]
```diff
@@ -72,19 +72,43 @@ SobolIndicesAlgorithm
 
 Examples
 --------
+
+Estimate first and total order Sobol' indices:
```
Comment (jschueller) [inline python/src/MartinezSensitivityAlgorithm_doc.i.in]: you need to skip a line here

File: python/src/SobolIndicesExperiment_doc.i.in [hunk 3]
```diff
@@ -51,13 +51,25 @@ SobolIndicesAlgorithm
 
 Examples
 --------
+
+Create a sample suitable to estimate first and total order Sobol' indices
```
Comment (jschueller) [inline python/src/SobolIndicesExperiment_doc.i.in]: you need to skip a line here

File: python/src/SobolIndicesExperiment_doc.i.in [hunk 4]
```diff
@@ -51,13 +51,25 @@ SobolIndicesAlgorithm
...
 >>> model = ot.SymbolicFunction(['X1', 'X2', 'X3'], formula)
->>> distribution = ot.ComposedDistribution([ot.Uniform(-1.0, 1.0)] * 3, \
-...                                         ot.IndependentCopula(3))
+>>> distribution = ot.ComposedDistribution([ot.Uniform(-1.0, 1.0)] * 3)
 >>> size = 10
->>> experiment = ot.SobolIndicesExperiment(distribution, size, True)
+>>> experiment = ot.SobolIndicesExperiment(distribution, size)
+>>> sample = experiment.generate()
+
+Create a sample suitable to estimate first, total order and second order Sobol' indices
```
Comment (jschueller) [inline python/src/SobolIndicesExperiment_doc.i.in]: you need to skip a line here

Comment (mbaudin47) [reply python/src/SobolIndicesExperiment_doc.i.in]: OK

File: python/src/SobolIndicesExperiment_doc.i.in (hunk 3 above)
Comment (mbaudin47) [reply python/src/SobolIndicesExperiment_doc.i.in]: OK

File: python/src/SaltelliSensitivityAlgorithm_doc.i.in [hunk 5]
```diff
@@ -67,17 +67,15 @@ Examples
...
-...                                         ot.IndependentCopula(3))
+>>> distribution = ot.ComposedDistribution([ot.Uniform(-1.0, 1.0)] * 3)
 >>> # Define designs to pre-compute
 >>> size = 100
->>> inputDesign = ot.SobolIndicesExperiment(distribution, size, True).generate()
+>>> inputDesign = ot.SobolIndicesExperiment(distribution, size).generate()
 >>> outputDesign = model(inputDesign)
 >>> # sensitivity analysis algorithm
->>> sensitivityAnalysis = ot.SaltelliSensitivityAlgorithm(inputDesign, outputDesign, size)
+>>> sensitivityAnalysis = ot.SaltelliSensitivityAlgorithm(inputDesign, outputDesign)
```
Comment (jschueller) [inline python/src/SaltelliSensitivityAlgorithm_doc.i.in]: size argument is missing

**Comment by sofianehaddad** [comment]
please drop 8ce2eaa commit

**Pull request merged**